.github/workflows/build.yml

```diff
@@ -141,3 +144,11 @@ jobs:
         with:
           name: build-scan-test-${{ matrix.java }}
           path: ~/.gradle/build-scan-data
+      - name: Archive Thread Dumps
```

Comment:
Maybe the artifact url generated by this task should be passed to "Parse JUnit tests". Than can enable "Parse JUnit tests" to add url to summary (like junit reports)